be/src/cloud/cloud_meta_mgr.cpp

```diff
@@ -447,6 +481,7 @@ Status CloudMetaMgr::sync_tablet_rowsets(CloudTablet* tablet, bool warmup_delta_
                         .tag("partition_id", tablet->partition_id())
                         .tag("tried", tried)
                         .tag("sleep", duration_ms);
+                proxy->set_unhealthy();
```

Comment:
why not `set_unhealthy()` right after we get `if (cntl.Failed())`? it should not rely on retry_times

Reply:
> why not `set_unhealthy()` right after we get `if (cntl.Failed())`? it should not rely on retry_times

done